src/Stryker.Core/Stryker.Core/MutationTest/MutationTestProcess.cs

```diff
@@ -79,12 +85,12 @@ public void Mutate()
                     path: file.FullPath,
                     options: cSharpParseOptions);
 
-                    _logger.LogDebug($"Mutating {file.Name}");
+                _logger.LogDebug($"Mutating {file.Name}");
                 // Mutate the syntax tree
                 var mutatedSyntaxTree = _orchestrator.Mutate(syntaxTree.GetRoot());
                 // Add the mutated syntax tree for compilation
                 mutatedSyntaxTrees.Add(mutatedSyntaxTree.SyntaxTree);
-
+                _logger.LogTrace($"Mutated {file.Name}:{Environment.NewLine}{mutatedSyntaxTree.ToFullString()}");
```

Comment:
Do we need this extra logging? I'm afraid that logfiles from large projects will get even more unclear thanks to these enormous blocks of code. We already log quite some code during rollbacking. Do you think this will make things more clear?

Reply:
I think it would be beneficial be able to access mutated code for mutation error analysis.
On the other hand, when testing, I realised that Trace entries are present when you request debug level. I think it would be better if we have a specific switch for that. Maybe correlating that with Devmode.
What to you think?

Reply:
During development you could also get the mutated code using debugging right? A devmode switch could be better here. If you think it is beneficial we can keep it with a switch :+1:

Reply:
When logging to file, all logging is written to the file. This is intended behaviour. If you mean that you see trace logging in the console when requesting debug logging then that is a bug and that should not be happening.

Reply:
Ok, added devmode has a supplemental condition